proto/regen/bond/v1alpha1/types.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+package regen.bond.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/bond";
+
+// BatchInfo represents the high-level on-chain information for a bond.
+message BondInfo {
+
+  // id of the bond
+  string id = 1;
```

Comment:
How the `id` will be constructed? If this will be a sequence number, then uint64 should be sufficient.

Reply:
It is generated as UUID. It is not a number.

Reply:
OK, let's add a comment to the proto. BTW: why not using a sequence number? I sure uint64 is big enough.

Reply:
We could use sequence number. Would be better?